compendium/DeclarativeServices/src/manager/ComponentManagerImpl.hpp

```diff
@@ -201,11 +206,13 @@ namespace cppmicroservices
              * \return a shared_future<void> on which to wait for the asynchronous work to complete.
              */
             std::shared_future<void> PostAsyncEnabledToDisabled(
-                std::shared_ptr<cppmicroservices::scrimpl::ComponentManagerState>& currentState);
+                std::shared_ptr<cppmicroservices::scrimpl::ComponentManagerState>& currentState,
+                std::shared_ptr<std::atomic<bool>> asyncStarted);
 
           private:
             FRIEND_TEST(ComponentManagerImplParameterizedTest, TestAccumulateFutures);
-
+            std::unordered_map<std::shared_ptr<std::atomic<bool>>, std::shared_ptr<ActualTask>> taskMap;
+            std::unordered_map<std::shared_ptr<std::atomic<bool>>, std::shared_ptr<CMEnabledState>> enStateMap;
```

Comment:
variables need a comment and/or more descriptive names.

Reply:
fixed